src/applications/disability-benefits/all-claims/validations.js

```diff
@@ -157,3 +158,35 @@ export const isValidYear = (err, fieldData) => {
     err.addError('The year can’t be in the future');
   }
 };
+
+export function startedAfterServicePeriod(err, fieldData, formData) {
+  const { serviceInformation } = formData;
+  if (
```

Comment:
I admit this might be a bit paranoid

Reply:
Alternatively, you could do `if (!_.get('servicePeriods.length', formData.serviceInformation, false)) { return; }`. Wouldn't check that it's an array, but that should be alright.